sskittenapi/catapi/catapi/events/event_handlers.py

```diff
@@ -38,6 +40,23 @@ def handle_cat_created(data: dto.JSON) -> None:
     logger.info(f"[{event_id}] Cat {cat_id} has been created")
     # TODO: Handle the async postprocessing of a created Cat here.
 
+    cat_partial_update = dto.PartialUpdateCat(url="http://placekitten.com/200/300")
+    cat_id = dto.CatID(data["cat_id"])
+    print("Iam", cat_partial_update)
+    loop = asyncio.get_event_loop()
+    print("loop", loop)
+    coroutine = cat_domain.partial_update_cat(
+        partial_update_cat=cat_partial_update,
+        cat_id=cat_id,
+    )
+    print("Iaminloop", coroutine)
+    loop.run_until_complete(coroutine)
+    print("Iamoutloop", coroutine)
+    if not coroutine:
```

Comment:
[question] Will `coroutine` be `None` even if no cat can be found?
If no cat is existing, what will actually happen and why?

Reply:
### Yes, you are right coroutine will never be none. I am going to remove this not-needed error handling.